Modules/_ssl.c

```diff
@@ -103,7 +103,13 @@ struct py_ssl_library_code {
 };
 
 /* Include generated data (error codes) */
+#if (OPENSSL_VERSION_NUMBER >= 0x30000000L)
```

Comment:
The main branch also has a code path for OpenSSL 3.1:

```c
#if (OPENSSL_VERSION_NUMBER >= 0x30100000L)
#include "_ssl_data_31.h"
```

Fedora 39 already has openssl-3.1.1. Do we need to care here?

Reply:
From https://github.com/python/cpython/commit/ede89af605b1c0442353435ad22195c16274f65d the commit message mentions: "backports of this prior to 3.12 will not include the openssl 3.1 header."

So I think we can skip this part.

Reply:
Ok, thanks for checking.